.circleci/config.yml

```diff
@@ -30,9 +30,26 @@ jobs:
             # Test
             - run: echo "Testing ..."
             - run: pytest --cov . --cov-branch . --cov-report term-missing --cov-fail-under 100
+    rust_build_format_test:
+        docker:
+            - image: cimg/rust:1.57.0
+        steps:
+            - checkout
+            # Format
+            - run: echo "Checking formatting ..."
+            - run: rustfmt --check ./implementations/rust/**/*.rs
+            # Build
+            - run: echo "Building ..."
+            - run: rustc ./implementations/rust/sieve/sieve_run.rs -o ./implementations/rust/sieve/sieve_run
+            - run: ./implementations/rust/sieve/sieve_run
+            # Test
+            - run: echo "Testing ..."
+            - run: rustc --test ./implementations/rust/sieve/sieve_test.rs -o ./implementations/rust/sieve/sieve_test
+            - run: ./implementations/rust/sieve/sieve_test
```

Comment:
Need to find a way to automate this. Most likely will require a bash script for each language to be executed instead of raw commands.